### DISCUSSION TITLE
Filter for file type is not working

### DISCUSSION THREAD

**Discussion by manjunathreddy1** [microsoft-search/pnp-modern-search#3283] [Q&A]
Hi All,

I have created Microsoft search filter for resource.listitem.fields.filetype to show file types search results documents and I have created search query to show only documents. But in filter no values are showing up. Please advice how we can fix this issue.

{searchTerms} IsDocument:true

**Comment by kasperbolarsen** [comment]
You have ensured that the Filter web part points to the Results web part and the Results Web part points to the Refiner web part?

I have just checked using v 4.9.2 and it works just fine

**Reply by manjunathreddy1** [reply]
I have already selected reults webpart in filter webpart and refiner webpart in results but still filter is not working for file type even with 4.9 version. Are you using the same field (resource.listitem.fields.filetype) for filtering. Also, will it be possible to share the steps which you followed.

**Comment by kasperbolarsen** [comment]
I am just using filetype

![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/145afa11-5973-41ed-a5e8-741658849cea)


![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/c74477f1-f36b-402b-91ee-8cc7c83dda07)

**Reply by manjunathreddy1** [reply]
For me filetype field is showing up select fields drop down but it is not showing up in edit filters drop down. Only resource.listItem.fields.filetype is showing up and if I select this field no data is showing up.

**Reply by kasperbolarsen** [reply]
Have you tried typing "filetype" in the Filter field? MS Search does not provide the info about which properties are refinable so we can't populate that field. In general, you should expect that more fields in the solution over time will be type it yourself rather than selection.
